pillar.example

```diff
@@ -31,18 +31,19 @@ netweaver:
   # Specify the path to already extracted SWPM installer folder
   swpm_folder: your_swpm_folder_absolute_path
   # Or specify the path to the sapcar executable & SWPM installer sar archive, to extract the installer
-  # The sar archive will be extracted to path specified at swpm_extract_dir (optional, by default /sapmedia/NW/SWPM)
+  # The sar archive will be extracted to a subfolder SWPM, under nw_extract_dir (optional, by default /sapmedia/NW/SWPM)
   # Make sure to use the latest/compatible version of sapcar executable, and that it has correct execute permissions
   sapcar_exe_file: your_sapcar_exe_file_absolute_path
   swpm_sar_file: your_swpm_sar_file_absolute_path
-  swpm_extract_dir: location_to_extract_swpm_sar_absolute_path
+  nw_extract_dir: location_to_extract_nw_media_absolute_path
   sapexe_folder: your_sapexe_sar_folder_absolute_path
   # Folder where the installation files are stored. /tmp/swpm_unattended by default. Set None to use
   # SAP default folders (it will only work with ASCS and ERS).
   # This folder content will be removed before the installation so be extra careful!
   installation_folder: /tmp/swpm_unattended
   # DB/PAS/AAS instances require additional DVD folders like NW Export or HDB Client folder
-  additional_dvd:
+  # Provide the software folder or archives with additional SAP software needed to install netweaver
+  additional_dvds:
     - required_dvd1_folder_absolute_path
```

Comment:
We had this `absolute_path` description before, but it looks incorrect, actually the path is relative, right?

Reply:
Yes,  the dvd path currently used it relative to `netweaver_inst_media`

Reply:
You are right! Thanks for confirming